tiltak/src/main/kotlin/no/nav/amt/tiltak/tiltak/repositories/HentKoordinatorerForGjennomforingQuery.kt

```diff
@@ -12,35 +14,60 @@ class HentKoordinatorerForGjennomforingQuery(
 ) {
 
 	private val rowMapper = RowMapper { rs, _ ->
-		val fornavn: String = rs.getString("fornavn")
-		val mellomnavn: String? = rs.getString("mellomnavn")
-		val etternavn: String = rs.getString("etternavn")
-
-		if (mellomnavn != null) {
-			"$fornavn $mellomnavn $etternavn"
-		} else {
-			"$fornavn $etternavn"
-		}
-
+		Pair(rs.getUUID("gjennomforingId"), getKoordinatorName(rs))
 	}
 
 	private val sql = """
-		SELECT a.fornavn    AS fornavn,
+		SELECT aagt.gjennomforing_id AS gjennomforingId,
+			   a.fornavn    AS fornavn,
 			   a.mellomnavn AS mellomnavn,
 			   a.etternavn  AS etternavn
 		FROM arrangor_ansatt a
 				 INNER JOIN arrangor_ansatt_rolle aar on a.id = aar.ansatt_id
 				 INNER JOIN arrangor_ansatt_gjennomforing_tilgang aagt on aar.ansatt_id = aagt.ansatt_id
-		WHERE aagt.gjennomforing_id = :gjennomforingId
+		WHERE aagt.gjennomforing_id IN (:gjennomforingIds)
 		  AND aar.rolle = 'KOORDINATOR'
+		  AND aagt.gyldig_fra < CURRENT_TIMESTAMP
+		  AND aagt.gyldig_til > CURRENT_TIMESTAMP
 	""".trimIndent()
 
 	fun query(gjennomforingId: UUID): List<String> {
-		return template.query(
+		val koordinatorer = query(listOf(gjennomforingId))
+
+		return koordinatorer[gjennomforingId]
+			?: emptyList()
+	}
+
+	fun query(gjennomforingIds: List<UUID>): Map<UUID, List<String>> {
```

Comment:
Det burde være en sjekk her på om listen er tom før man kjører SQLen. 
Hvis man bruker `in()` med en tom liste så vil querien tryne istedenfor å returnere tom liste tilbake.
Har gått på den smellen før 😅

Reply:
Det har jeg og, når du sier det 👍  skal fikse!